clients/js/package.json

```diff
@@ -72,15 +72,19 @@
     "node": ">=14.17.0"
   },
   "dependencies": {
-    "isomorphic-fetch": "^3.0.0",
-    "cliui": "^8.0.1"
+    "cliui": "^8.0.1",
+    "isomorphic-fetch": "^3.0.0"
   },
   "peerDependencies": {
+    "@aws-sdk/client-bedrock-runtime": "^3.496.0",
     "@google/generative-ai": "^0.1.1",
     "cohere-ai": "^5.0.0 || ^6.0.0 || ^7.0.0",
     "openai": "^3.0.0 || ^4.0.0"
   },
   "peerDependenciesMeta": {
+    "@aws-sdk/client-bedrock-runtime": {
```

Comment:
Do you think we could benefit from adding a version range here like we have for `cohere` and `openai`?

Reply:
Do you mean this line? https://github.com/chroma-core/chroma/pull/1659/files#diff-f7c3370276357e7881ff5ffe05e8e4cdc98688ec828f38d4a436d8f6e9c9ad7dR79